pkg/controller/functionApi.go

```diff
@@ -369,3 +369,35 @@ func getContainerLog(kubernetesClient *kubernetes.Clientset, w http.ResponseWrit
 
 	return nil
 }
+
+// FunctionApiPodList: Get list of pods currently used by function
+func (a *API) FunctionApiPodList(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	name := vars["function"]
```

Comment:
Please check if we need any error handling here of "function" key not in vars.

Reply:
This function is registered as event handler for the following route 
`r.HandleFunc("/v2/functions/{function}/pods", api.FunctionApiPodList).Methods("GET")`

Unless and untill we have the route like this `/v2/functions/{function}/pods`, it will not hit the event handler `FunctionApiPodList` 

Similar code has been used for other eventhandlers like `FunctionApiDelete`, `FunctionApiGet`, `FunctionApiUpdate` and `FunctionPodLogs`

Reply:
Fixed